app/api/tool_utilities.py

```diff
@@ -1,8 +1,8 @@
 import json
 import os
-from app.services.logger import setup_logger
-from app.services.tool_registry import ToolFile
-from app.api.error_utilities import VideoTranscriptError, InputValidationError, ToolExecutorError
+from services.logger import setup_logger
```

Comment:
You must not need to remove the app folder because it allows to avoid dependency issues.